AppCenter/AppCenter/Internals/Channel/MSChannelGroupDefault.m

```diff
@@ -231,4 +236,10 @@ - (void)setMaxStorageSize:(long)sizeInBytes completionHandler:(nullable void (^)
   });
 }
 
+- (void)authTokenContext:(__unused MSAuthTokenContext *)authTokenContext didReceiveAuthToken:(NSString *)authToken {
```

Comment:
I'm a bit confused by the parameter name.  Does it mean, "did recieve _this_ auth token"?  To me it looks more like a method name which returns a bool

Reply:
The `didRecieveXXX` naming pattern comes from iOS APIs from Apple. 

For example, `application:didReceiveRemoteNotification:fetchCompletionHandler:` https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1623013-application?language=objc.

Reply:
I think it's a convention to name callbacks like that in objective c, but I agree that it can be confusing

Reply:
`willAction` and `didAction` is a classic in objective c.

Reply:
And yes you can see the first parameter as the sender of the event.